hdf5.spec

```diff
@@ -291,7 +286,9 @@ ln -s ../configure .
 %configure \
   %{configure_opts} \
```

Comment:
does this need also --enable-parallel for the parallel version of the library?

Reply:
I believe the --enable-parallel is on the configure line for building with mpi's; It is not on the java configure.